ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/ovn/sdn2ovn/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
```

Comment:
I guess you can replace installer team members with networking team members here?